### PR TITLE
Bump serving manifests

### PR DESCRIPTION
**Pull request by ReToCode**
## Proposed Changes
- Drops the index when downloading artifacts from Serving (https://github.com/openshift-knative/serving/pull/89)
- Run `make generated-files`

/cc @nak3

### CODE REVIEW

File: openshift-knative-operator/cmd/operator/kodata/knative-serving/1.7.0/2-serving-core.yaml [hunk 1]
```diff
@@ -6416,16 +6524,16 @@ spec:
...
+  - type: Resource
+    resource:
+      name: cpu
+      target:
+        type: Utilization
+        # Percentage of the requested CPU
+        averageUtilization: 100
 ---
 # Webhook PDB.
-apiVersion: policy/v1
```
[the remark this answers is not in the document]
Comment (nak3) [reply openshift-knative-operator/cmd/operator/kodata/knative-serving/1.7.0/2-serving-core.yaml:6428]: Yeah, we need to drop this patch https://github.com/openshift-knative/serving/blob/release-v1.7/openshift/patches/006-pdb.patch

https://github.com/openshift/knative-serving/pull/1290 should have been cherry-picked

[the remark this answers is not in the document]
Comment (nak3) [reply openshift-knative-operator/cmd/operator/kodata/knative-serving/1.7.0/2-serving-core.yaml:6428]: I opened https://github.com/openshift-knative/serving/pull/204

**Comment by ReToCode** [comment]
/hold

@skonto do you want to integrate this into your PR to avoid rebasing/regenerating manifests?

**Comment by skonto** [comment]
```
TestActivatorHAGraceful
````

File: openshift-knative-operator/hack/update-manifests.sh [hunk 2]
```diff
@@ -45,7 +45,7 @@ function download_serving {
     target_file="$target_dir/$index-$file"
 
     if [[ ${KNATIVE_SERVING_MANIFESTS_DIR} = "" ]]; then
-      url="https://raw.githubusercontent.com/openshift-knative/serving/${branch}/openshift/release/artifacts/$index-$file"
+      url="https://raw.githubusercontent.com/openshift-knative/serving/${branch}/openshift/release/artifacts/$file"
```
Comment (skonto) [inline openshift-knative-operator/hack/update-manifests.sh:48]: So we stop downloading the files with index and get the other ones?

Comment (ReToCode) [reply openshift-knative-operator/hack/update-manifests.sh:48]: There is no more index in the files in Serving from 1.7+.

**Comment by cardil** [comment]
Looks like this PR is replacing the https://github.com/openshift-knative/serverless-operator/pull/1988

**Review by cardil — APPROVED** [review]
/lgtm

**Pull request merged**